testing/run_tests.py

```diff
@@ -110,12 +110,20 @@ def RunEngineExecutable(build_dir, executable_name, filter, flags=[],
     print('Skipping %s due to filter.' % executable_name)
     return
 
-  executable = FindExecutablePath(os.path.join(build_dir, executable_name))
+  if IsLinux():
+    # Use unstripped executables in order to get better symbolized crash
+    # stack traces on Linux.
+    executable = os.path.join(build_dir, "exe.unstripped", executable_name)
+    # Some tests depend on the EGL/GLES libraries placed in the build directory.
+    env = {"LD_LIBRARY_PATH": build_dir}
```

Comment:
I wonder if we should just modify the build output directory for unstripped executable to just be next to stripped variants. That way, we won't have to do library path manipulations or extra flags. Perhaps just have a suffix to the executable name (`exe_name_unstripped`). This will also help in the cases where folks need to attach debuggers and such to the unstripped exe runs.

I suppose the Right™ way to do this would be have stripping be a target specific option but right now its all or nothing unfortunately.

Reply:
I would prefer to keep the current directory structure in order to maintain similarity to the original Chromium build rules (and other build rules like Dart's that might assume Chromium's setup)

Most of the engine tests can run directly out of `exe.unstripped` (in particular, the auto-generated  `GetFixturesPath` paths are fully qualified and do not depend on the location of the executable).  But the embedder tests that rely on SwiftShader EGL/GLES must have a way to locate those libraries.

Reply:
Sure. It's your call. LGTM either way.